### DISCUSSION TITLE
Windows Found a Virus "Win32/FRProxy upon installing this Kohya git

### DISCUSSION THREAD

**Discussion by turtlkky** [bmaltais/kohya_ss#511] [General]
Appears when you perform the install "\kohya_ss\venv\Lib\site-packages\gradio\frpc_windows_amd64_v0.2" is installed.

Windows is detecting this as a virus and recommends removal of this Fast Reverse Proxy as it is know as a hacker tool that exploits commercial license hacks.

Is this intended to be part of the Kohya installation?  If not, please inspect your installation processes.

**Comment by oliverban** [comment]
Now you made me scared, I just installed kohya and all is working, just trained my first Lora! Any words from the devs on this? What is that Fast Reverse Proxy used for?

**Reply by BarfingLemurs** [reply]
https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Troubleshooting#--share-non-functional-after-gradio-322-update
